tests/api/test_payment.py

```diff
@@ -1,15 +1,21 @@
 from unittest.mock import patch
 
 import graphene
+import pytest
+from tests.api.utils import get_graphql_content
```

Comment:
Relative import needed here

Reply:
Those are tests and this util is imported everywhere like this.